packages/@tinacms/graphql/src/resolver/media-utils.ts

```diff
@@ -23,19 +24,28 @@ import { GraphQLConfig } from '../types'
 
 export const resolveMediaCloudToRelative = (
   value: string,
-  config: GraphQLConfig = { useRelativeMedia: true }
+  config: GraphQLConfig = { useRelativeMedia: true },
+  schema: TinaCloudSchemaEnriched
 ) => {
   if (config && value) {
     if (config.useRelativeMedia === true) {
       return value
-    } else {
+    }
+
+    if (isValidTinaSchema(schema) === true) {
```

Comment:
This name was a little confusing at first. Maybe it could be renamed to something that describes what it is doing a bit better?
Maybe
- hasSchemaMediaConfig ?
- isValidSchemMediaConig ?

Reply:
Agreed.  Changed to `hasTinaMediaConfig` 👍

Reply:
awesome. When that is cleaned up we can merge it.